cmd/clean/clean.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/buildsafe/bsf/cmd/init"
+	"os"
+)
+
+func init() {
+	rootCmd.AddCommand(cleanCmd)
```

Comment:
This logic should go over here:
https://github.com/buildsafedev/bsf/blob/a2c73d1a07c03290df94270b30c687bac5491b41/cmd/cmd.go#L70